src/en/authors-charm-config.md

```diff
@@ -2,25 +2,26 @@ Title: Creating config.yaml and configuring charms
 
 # Charm configuration
 
-The optional `config.yaml` file defines how a service running the charm can be
-configured by its user.
+The optional `config.yaml` file defines how the charm can be configured by the
+user.
```

Comment:
Do you think it's worthwhile here making a distinction between different users? Eg. here it'd be a "charm author" which is different to "a user that wishes to deploy this charm".

Reply:
I chose "by the user" carefully. This, to me, and hopefully the reader, indicates the end user. I had assumed the charm author knew what config.yaml is for. However, after looking at other pages and where this page fits in, I realized this may not be the case.

This page is not found in the menu. Actually, it is quite far down the rabbit hole:

https://jujucharms.com/docs/2.2/authors-intro
https://jujucharms.com/docs/2.2/authors-charm-components
https://jujucharms.com/docs/2.2/authors-charm-config

I'll reword. But it's subtle because this file is available to the end user if they choose to download and inspect the charm, for the exact purpose of discovering how it can be configured, and this is the file that shows that to them, the end user.